test/CTestList.cmake

```diff
@@ -99,6 +99,8 @@ endfunction(add_test_u)
 #=============================================================================
 add_test_r(test_1 4)
 add_test_r(test_2 4)
+add_test_r(test_tgv 4)
```

Comment:
Probably a nitpick, but we can drop the `test_` prefix. Instead call the tests `tgv_mol` and `tgv_godunov`.